src/index.scss

```diff
@@ -27,41 +27,46 @@
 
 .homePageContent {
   display: flex;
-}
 
-.homePageContent .sideNavBar {
-  flex: 20%;
-  border-style: solid;
-  margin: 20px;
-  text-align: center;
-}
+  .sideNavBar {
+    width: 20%;
+    border-style: solid;
+    margin: 20px;
+    text-align: center;
+    padding-top: 20px;
+  }
 
-.homePageContent .flightsTable {
-  flex: 80%;
-  border-style: solid;
-  margin: 20px;
-  text-align: center;
+  .flightsTable {
+    width: 80%;
+    border-style: solid;
+    margin: 20px;
+    padding-top: 20px;
+    text-align: center;
+  }
 }
 
 .topNavBar {
   padding: 20px;
-  align-items: center;
+  align-items: right;
   display: flex;
-  margin: 0 auto;
+  margin: 0 88%;
   max-width: 600px;
-}
 
-.topNavBar .links {
-  margin-left: auto;
-  padding: 6px;
-}
+  .links {
+    margin-left: 100%;
+    padding: 6px;
+  }
 
-.topNavBar a {
-  margin-left: 16px;
-  padding: 6px;
-  color: #36454F;
-}
+  a {
+    margin-left: 16px;
+    padding: 6px;
+    color: #36454F;
+  }
 
-.topNavBar a:hover {
-  color: red
+  a:hover {
```

Comment:
you could move this up to the `a { ... }` block.

```
  a {
    margin-left: 16px;
    padding: 6px;
    color: #36454F;
    
    &:hover {
        color: red;
    }
}
```

Reply:
Ah! Yes! Will keep that &: syntax in the back of my mind - thank you!